### ISSUE TITLE
Any fix for window/menu background flickering?

### ISSUE THREAD

**Issue by Kinzushi** [skotlex/ffxiv-material-ui#236]
Fresh install of Material UI for FFXIV, both Discord and Black styles have window/menu flickering. It is very inconsistent and random, but I am reliably getting it on the marketboard menu window in particular. 

- FPS is uncapped
- 4K HD UI resolution is **turned off**
- Not using any other add-ons or shaders
- Installed via FFXIV TexTools 
- Playing on 1080p Windowed Borderless, **Full Screen did not fix it**
- Nvidia graphics card, 1070

Any fixes?

**Comment by skotlex** [comment]
#229 duplicate

**Comment by Anseilon** [comment]
I've found that, at least on my end, that lowering the cap on FPS in-game seems to mitigate the flickering (for me, on 144 Hz, setting it to half refresh rate, 72 Hz).  HD, TexTools installation, 1080p, but capping at 144 or just leaving it uncapped would cause inconsistent flickering.  I'm on a 3060 Ti, on 471.68.  No earlier driver seems to correct the behavior.

I've also found that capping via NVCP doesn't help in any way.  At least for me.
